### ISSUE TITLE
Upload image closes when I press enter

### ISSUE THREAD

**Issue by uglyeoin** [joomla/joomla-cms#24943]
### Steps to reproduce the issue
Go to upload an image from an article, I used the image button to produce the popup click on "upload" click "browse" select an image and hit enter.

### Expected result
The image will be available and I can press the upload button to upload the image

### Actual result
The window closes.  

I can go through the same process and click the "open" button with my mouse and things work as expected.  This is purely if I use the keyboard, and therefore, an accessibility problem.

I can double click on the file and things work as expected.  

=============
System Information
=============
php: Linux squareballoon.gds.guru.net.uk 3.10.0-714.10.2.lve1.5.19.3.el7.x86_64 #1 SMP Tue Aug 7 21:33:29 EDT 2018 x86_64
dbserver: mysql
dbversion: 5.7.26
dbcollation: utf8_general_ci
dbconnectioncollation: utf8mb4_general_ci
phpversion: 7.2.18
server: LiteSpeed
sapi_name: litespeed
version: Joomla! 3.9.6 Stable [ Amani ] 7-May-2019 15:00 GMT
platform: Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:66.0) Gecko/20100101 Firefox/66.0

### Additional comments
I suppose it's possible this is a browser issue, but even so it's one we should try to overcome.

**Comment by ghost** [comment]
can you please post the System Information in the post so People haven't to download them.

**Comment by uglyeoin** [comment]
> 
> 
> can you please post the System Information in the post so People haven't to download them.

I have done so but it's made it a very long post.  Is that what you meant?  Should I edit it to remove most of that stuff except the key System Information?

**Comment by uglyeoin** [comment]
I'm sure that can be the only sensible solution I have changed.

**Comment by ghost** [comment]
thanks @uglyeoin for Info so Dev. can read easy technical Informations.

**Comment by brianteeman** [comment]
I couldnt replicate this.

> I suppose it's possible this is a browser issue, but even so it's one we should try to overcome.

Which browser are you using?

**Comment by uglyeoin** [comment]
Chrome

**Comment by brianteeman** [comment]
I was testing using Chrome Version 74.0.3729.157 (Official Build) (64-bit) for windows

**Comment by uglyeoin** [comment]
Weird. Let me try again and make a video to ensure we're both doing the same thing

**Comment by uglyeoin** [comment]
Sorry it was Firefox, I noticed in my system information that it mentioned the browser.  Here is a video of me using my mouse most of the way and then hitting enter.  It seems to close both windows and doesn't give me the "upload" option because of this

![imagesthenhitreturn](https://user-images.githubusercontent.com/4665225/58037411-4b5e7a00-7b25-11e9-851a-5861349cfec6.gif)

**Comment by brianteeman** [comment]
Your video shows that you are not using the JCE editor for images. Please close this here and report it to JCE - it is nothing to do with the core of Joomla

**Comment by alikon** [comment]
not from core<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24943">issues.joomla.org/tracker/joomla-cms/24943</a>.</sub>
